client/app/scripts/liveblog-syndication/actions/incoming-syndication.js

```diff
@@ -27,6 +27,23 @@ liveblogSyndication
                             });
                         });
                     });
+                },
+                publish: function(post) {
+                    postsService.savePost(post.blog, post, undefined, {post_status: 'open'})
+                        .then(function(post) {
+                            Dispatcher.dispatch({
+                                type: 'ON_YANKED_POST',
```

Comment:
Not sure what 'yanked' means here, or why it's being used for two quite different actions. For the sake of readability, I would have 2 different types (e.g. ON_SAVED_POST and ON_REMOVED_POST) which can both be handled by the same code in the listener